### ISSUE TITLE
Add `aria-hidden=false` and `role='button'` to focusable icon in `ToolCard`

### ISSUE THREAD

**Issue by ssciolla** [tl-its-umich-edu/canvas-app-explorer#260]
From the accessibility team:

Issue: The control that invokes the tooltip is an <svg> that admits focus but has an aria-hidden=“true” attribute. Normally <svg> and font icons that are decorative should have that attribute, but not when they are informational.

To Fix: Remove the aria-hidden attribute and might as well give it a role=“button” since that is what it is, really.
